src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -287,31 +300,28 @@ def get_accounts(self, protected_ou_ids=None, include_root=True):
     def get_organization_info(self):
         response = self.client.describe_organization()
         return {
-            "organization_master_account_id": response.get(
-                'Organization').get(
-                    'MasterAccountId'
-                ),
-            "organization_id": response.get('Organization').get('Id'),
-            "feature_set": response.get('Organization').get('FeatureSet')
+            "organization_master_account_id": response.get("Organization").get(
+                "MasterAccountId"
+            ),
```

Comment:
For readability, could you change this multi-liner into:
```suggestion
            "organization_master_account_id": (
                response
                .get("Organization")
                .get("MasterAccountId")
            ),
```